runners/google-cloud-dataflow-java/worker/build.gradle

```diff
@@ -93,22 +92,26 @@ dependencies {
   compile library.java.joda_time
   shadow library.java.vendored_guava_26_0_jre
   compile library.java.slf4j_api
+  compile "com.google.api.grpc:proto-google-common-protos:1.17.0"
+  permitUnusedDeclared "org.conscrypt:conscrypt-openjdk:1.1.3:linux-x86_64"
   compile "javax.servlet:javax.servlet-api:3.1.0"
-  compile "org.conscrypt:conscrypt-openjdk:2.5.1:linux-x86_64"
+  //compile "org.conscrypt:conscrypt-openjdk:2.5.1:linux-x86_64"
   compile "org.eclipse.jetty:jetty-server:9.2.10.v20150310"
   compile "org.eclipse.jetty:jetty-servlet:9.2.10.v20150310"
-  compile library.java.error_prone_annotations
+  compile "com.google.auth:google-auth-library-credentials:0.19.0"
+  compile "com.google.guava:guava:25.1-jre"
+  compile "io.opencensus:opencensus-api:0.24.0"
+  compile "org.checkerframework:checker-qual:2.0.0"
+  compile "org.conscrypt:conscrypt-openjdk-uber:2.2.1"
   compile library.java.slf4j_jdk14
-
+  permitUnusedDeclared library.java.slf4j_jdk14
```

Comment:
nitpick - there should be a be a blank line about the comment - like there was before.